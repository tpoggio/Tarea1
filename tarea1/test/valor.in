asociar 15 4.0
esClave 15
valor 15
asociar 95 24.0
esClave 95
valor 95
asociar 85 2.0
asociar 65 3.0
asociar 25 67.0
asociar 35 -12.0
asociar 75 2.5
asociar 45 2.1
asociar 55 -24.0
asociar 85 32.0
valor 85
asociar 5 10.0
valor 5
asociar 1 10.0
esClave 1
Fin