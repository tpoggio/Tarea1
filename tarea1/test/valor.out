1>Asociar.
2>15 es clave. 
3>15 est치 asociado a 4.00. 
4>Asociar.
5>95 es clave. 
6>95 est치 asociado a 24.00. 
7>Asociar.
8>Asociar.
9>Asociar.
10>Asociar.
11>Asociar.
12>Asociar.
13>Asociar.
14>Asociar.
15>85 est치 asociado a 2.00. 
16>Asociar.
17>5 est치 asociado a 10.00. 
18>Asociar.
19>1 no es clave. 
20>Fin.
